venia-integration-tests/run-tests.js

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env node
+const glob = require('glob');
+const { exec } = require('child_process');
+
+var argv = require('yargs/yargs')(process.argv.slice(2))
+    .option('url', {
+        alias: 'baseUrl',
+        demandOption: true,
+        describe: 'URL to run tests against',
+        type: 'string',
+        nargs: 1
+    })
+    .option('p', {
+        alias: 'parallel',
+        default: 1,
+        describe: 'Number of parallel runs',
+        type: 'number',
+        nargs: 1
+    })
+    .option('u', {
+        alias: 'update',
+        default: false,
+        describe: 'Update snapshots',
+        type: 'boolean'
+    })
+    .option('s', {
+        alias: 'spec',
+        default: null,
+        describe: 'Spec files to run',
+        type: 'string',
+        nargs: 1
+    })
+    .option('h', {
+        alias: 'help',
+        describe: 'Show run-tests help'
+    })
+    .help('h', 'Show run-tests help')
+    .version(false).argv;
+
+const { baseUrl, parallel: parallelRuns, update, spec } = argv;
+
+if (!baseUrl) {
+    console.error(
+        'Missing baseUrl. Please provide a baseUrl using the --baseUrl arg'
+    );
+
+    return;
+}
+
+const files = spec ? spec.split(',') : glob.sync('./src/tests/**/*.spec.js');
+
+if (files.length < parallelRuns) {
+    console.error('Can not have more parallel runs than tests.');
+
+    return;
+}
+
+const testsPerRun = files.length / parallelRuns;
+const dockerRuns = {};
+
+const port = new URL(baseUrl).port;
+
+let dockerCommand = null;
+
+if (port) {
+    // run docker on local instance
+    console.log(`Running tests on local instance ${baseUrl}`);
+
+    dockerCommand = `docker run --rm -v ${
+        process.env.PWD
+    }:/venia-integration-tests -w /venia-integration-tests --entrypoint=cypress cypress/included:8.3.1 run --browser chrome --config baseUrl=https://host.docker.internal:${port},screenshotOnRunFailure=false --config-file cypress.config.json --env updateSnapshots=${update} --headless`;
```

Comment:
This doesn't need `--network host` like in the previous script?

Reply:
Nah it doesn't need that any more 😄